html/manifest/id.json

```diff
@@ -12,9 +12,11 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": "95"
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": false
             },
```

Comment:
```suggestion
            "firefox_android": "mirror",
```

Reply:
In this case, I verified that we do not parse the `id` in Android after Gecko is done with it, so I am going to leave this as `false` while adding the notes to only firefox.